commands/fetch.go

```diff
@@ -14,14 +14,22 @@ type Fetch struct {
 }
 
 func (cmd *Fetch) Command() *imap.Command {
-	items := make([]interface{}, len(cmd.Items))
-	for i, item := range cmd.Items {
-		items[i] = imap.RawString(item)
-	}
+	// Handle FETCH macros separately as they should not be serialized within parentheses
+	if len(cmd.Items) == 1 && (cmd.Items[0] == imap.FetchAll || cmd.Items[0] == imap.FetchFast || cmd.Items[0] == imap.FetchFull) {
+		return &imap.Command{
+			Name:      "FETCH",
+			Arguments: []interface{}{cmd.SeqSet, cmd.Items},
```

Comment:
Shouldn't this be `imap.RawString(cmd.Items[0])`?

Reply:
Good catch! Fixed.